examples/CockroachEFCoreExample.csproj

```diff
@@ -16,9 +16,13 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="8.0.0" />
+      <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" />
       <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL.NetTopologySuite" />
       <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL.NodaTime" />
     </ItemGroup>
 
+
+    <ItemGroup>
+      <ProjectReference Include="..\src\EFCore.PG.CockroachDB\EFCore.PG.CockroachDB.csproj" />
```

Comment:
let's wait for the package to be available on nuget. i don't think it will be good for the main docs for this to guide people towards building from source and using that

Reply:
Fixed.